multilingual-tools.php

```diff
@@ -5,7 +5,7 @@
 Description: Set of tools to test themes and plugins multilingual compatibility.
 Author: OnTheGoSystems
 Author URI: https://www.onthegosystems.com/
-Version: 2.2.4
+Version: 2.2.5
 */
 
 define('WPML_CTT_VERSION', '2.2.4');
```

Comment:
Oops @diiegopereira 

I just noticed the version number missing here:
```suggestion
define('WPML_CTT_VERSION', '2.2.5');
```

I will go ahead and apply it myself.